tests/integration/simple-app.test.ts

```diff
@@ -22,6 +24,8 @@ import {
   startMockBlobStore,
 } from '../utils/helpers.js'
 import { nextVersionSatisfies } from '../utils/next-version-helpers.mjs'
+import { purgeCache } from '@netlify/functions'
+import { afterEach } from 'node:test'
```

Comment:
🤔 shouldn't this be from `vitest` like all the others?

Reply:
oooh, yes

Reply:
https://github.com/opennextjs/opennextjs-netlify/pull/2727/commits/4a02e345b378a22fec355ced93a1e9ffa9886a87